src/ui/components/atoms/label-articulo/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.label {
+    background: $ads-background-main;
+    padding: 1.26rem;
+    /*text-align: center;*/
+    width: 100%;
+
+    @include respond-to('medium') {
+        width: 80%;
+    }
+    
+    @include respond-to('large') {
+        width: 30%;
+    }
+    
+    @include respond-to('extra-large') {
+        width: 30%;
+    }
+}
+
+.label__text {
+    color: $ads-primary-main;
+    font-family: Montserrat;
```

Comment:
Nice please check if the font was correctly embedded in the project if it is not please do it